compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/layout/Layout.kt

```diff
@@ -134,6 +134,16 @@ inline fun Layout(
     )
 }
 
+/*
+ * TODO: This is a version of Layout not using SAM-converted [MeasurePolicy] interface.
+ * Remove me, when SAM conversion for [MeasurePolicy] works again.
+ */
+@Composable inline fun Layout(
+    content: @Composable () -> Unit,
+    modifier: Modifier = Modifier,
+    crossinline measurePolicy: MeasureScope.(List<Measurable>, Constraints) -> MeasureResult
+) = Layout(content, modifier, MeasurePolicy { x, y -> measurePolicy(x, y) })
```

Comment:
We also had another [function](https://github.com/JetBrains/androidx/commit/69f076da237d0e57de702bef9420b6f37d29f014). Do we need it as well?

Reply:
Yes, good point. Now I'll take a look.

Reply:
I checked the tests and samples that are described here https://github.com/JetBrains/compose-jb/blob/d550f2763f741b9636d92b63a97ffb451e6663b1/compose/README.md#run-tests

I think only one Layout function is enough